drift/lib/src/runtime/query_builder/schema/table_info.dart

```diff
@@ -28,6 +28,14 @@ mixin TableInfo<TableDsl extends Table, D> on Table
   @override
   Set<Column> get primaryKey => $primaryKey;
 
+  /// The unique key of this table. Can be empty if no custom primary key has
+  /// been specified.
+  ///
+  /// Additional to the [Table.primaryKey] columns declared by an user, this
+  /// also contains auto-increment integers, which are primary key by default.
+  @override
+  List<Set<Column>> get uniqueKeys => const [];
```

Comment:
It could be a `List<Set<GeneratedColumn>>` in this override though, right?

Reply:
Really